io.openrct2.OpenRCT2.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: io.openrct2.OpenRCT2
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: openrct2
+rename-appdata-file: openrct2.appdata.xml
+rename-desktop-file: openrct2.desktop
+rename-icon: openrct2
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Pulseaudio access
+  - --socket=pulseaudio
+  # Filesystem access
+  # to access local images and other files
+  - --filesystem=home
+  # Network access
+  # to access remote images
+  - --share=network
+  # OpenGL access
+  - --device=dri
+modules:
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/SDL/SDL_ttf-2.0.11.json
+  - libzip.yaml
+  - Jansson.yaml
+  - SpeexDSP.yaml
+  - OpenRCT2.yaml
```

Comment:
Any reason for splitting each module on it's own file?

Reply:
I was mostly following the example of [another app](https://github.com/flathub/app.rednotebook.RedNotebook) I've looked at recently. Though I did also think it would make it easy to move some of these (e.g. `libzip`) to `shared-modules` later.

I'm happy enough to change it if the preference is to stick things together in a single file.

Reply:
By convention the modules are split into different files if they are generated automatically by a script, but whatever works for you.

Reply:
Ooh, are there handy scripts for generating things like that automatically? That could save a bit of time.

Reply:
https://github.com/flatpak/flatpak-builder-tools has a bunch of tools for different language buildsystems. patches welcome!

Reply:
@takluyver  While it's ultimately your choice, I strongly prefer self-containted manifests instead of splitting modules to separate files.

Reply:
I have no preference, so if you have a strong one, I'll follow that. :-)